blueprint/api.apib

```diff
@@ -169,6 +169,14 @@ You can list all comments or retrieve individual comments, as well as create and
 
     + Attributes (Comment Response)
 
++ Response 401 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (JSON Web Token Invalid Response)
+
++ Response 403 (application/vnd.api+json; charset=utf-8)
+
+    + Attributes (Forbidden Response)
```

Comment:
So `create` does a `load_and_authorize_changeset`, right? What does that tell you about the responses you need to document and test?